lib/workos/client.rb

```diff
@@ -108,11 +108,15 @@ def handle_error_response(response:)
           request_id: response['x-request-id'],
         )
       when 422
-        errors = json['errors'].map do |error|
-          "#{error['field']}: #{error['code']}"
-        end.join('; ')
-
-        message = "#{json['message']} (#{errors})"
+        errors = nil
+        if json['errors']
+          errors = json['errors'].map do |error|
+            "#{error['field']}: #{error['code']}"
+          end.join('; ')
+        end
+
+        message = json['message']
+        message += " (#{errors})" if errors
```

Comment:
This is perfectly fine as is. If you're feeling fancy you could do 

```rb
[message, errors].compact.join(' ')
```

Example:

```
irb(main):001:0> foo = "bar"
=> "bar"
irb(main):002:0> baz = nil
=> nil
irb(main):003:0> [foo, baz].compact.join(' ')
=> "bar"
irb(main):004:0> baz = "bits"
=> "bits"
irb(main):005:0> [foo, baz].compact.join(' ')
=> "bar bits"
```

compact will remove nil items from the array.

Reply:
Nice, thanks!

Reply:
- [ ] Update message to use `[message, errors].compact.join(' ')`